packages/subgraph/src/plugin/adminMembers.ts

```diff
@@ -0,0 +1,79 @@
+import {Administrator, AdministratorAdminPlugin} from '../../generated/schema';
+import {
+  Granted,
+  Revoked,
+} from '../../generated/templates/AdminMembers/PermissionManager';
+import {generateAdministratorAdminPluginEntityId} from '../utils/ids';
+import {
+  generateEntityIdFromAddress,
+  generatePluginEntityId,
+} from '@aragon/osx-commons-subgraph';
+import {dataSource, store} from '@graphprotocol/graph-ts';
+
+export function handleGranted(event: Granted): void {
+  if (
+    isCorrectEvent(
+      event.params.permissionId.toHexString(),
+      event.params.where.toHexString()
+    )
+  ) {
+    const pluginAddress = event.params.where;
+    const administratorAddress = event.params.who;
+    const pluginEntityId = generatePluginEntityId(pluginAddress);
+    const administratorEntityId =
+      generateEntityIdFromAddress(administratorAddress);
+    let administrator = Administrator.load(administratorEntityId);
+    if (!administrator) {
+      administrator = new Administrator(administratorEntityId);
+      administrator.address = administratorEntityId;
+      administrator.save();
+    }
+
+    const administratorMappingId = generateAdministratorAdminPluginEntityId(
```

Comment:
naming a bit confusing here. line 23 calls a generateEntityIdFromAddress and the variable is called `administratorEntityId` then in this line we call `generateAdministratorAdminPluginEntityId` but the variable is called `administratorMappingId`

Are these different?
Should we be calling both with `generateAdministratorAdminPluginEntityId`?
If they are different, can we name them a bit more clearly?

Reply:
The `administratorEntityId` corresponds to `Administrator` entity, it should be the admin address (`generatePluginEntityId`). 

The `administratorMappingId` corresponds to `AdministratorAdminPlugin`entity, the many-to-many relationships between the `Administrator` and the `AdminPlugin`,  it should be the concatenation of the `AdminPluginId` and the `AdministratorId` (`generateAdministratorAdminPluginEntityId`)

I changed the variable names to be more clear,  `administratorMemberEntityId` for the `administratorEntityId` and `administratorAdminMappingId` for the `administratorMappingId` but it still can be confusing due to the similarity of the names `administrator` and `admin`. 

wdyt?

Reply:
followinf task https://aragonassociation.atlassian.net/browse/OS-1155